packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -577,13 +577,17 @@ class EditorPage {
 const blockNames = {
 	paragraph: 'Paragraph',
 	gallery: 'Gallery',
+	address: 'Address',
 	columns: 'Columns',
+	contactInfo: 'Contact Info',
 	cover: 'Cover',
+	email: 'Email Address',
 	heading: 'Heading',
 	image: 'Image',
 	latestPosts: 'Latest Posts',
 	list: 'List',
 	more: 'More',
+	phone: 'Phone Number',
```

Comment:
Could these changes  which appear Jetpack-specific be moved to gutenberg-mobile?